### PR TITLE
Show active users

### PR DESCRIPTION
**Pull request by Josipmrden**
### Description

`SHOW ACTIVE USERS INFO` is a command that displays information about users currently logged into Memgraph and session information about them.

### Pull request type

Please check what kind of PR this is:

- [X] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
https://github.com/memgraph/memgraph/pull/2183

### Checklist:

- [X] Check all content with Grammarly
- [X] Perform a self-review of my code
- [X] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [X] The build passes locally
- [X] My changes generate no new warnings or errors
- [X] Add a corresponding label
- [X] If release-related, add a product and version label
- [X] If release-related, add release note on product PR

### CODE REVIEW

**Review by kgolubic — APPROVED** [review]
Everything looks OK to me.

**Comment by Josipmrden** [comment]
Addressed @kgolubic

File: pages/database-management/server-stats.md [hunk 1]
```diff
@@ -157,3 +156,18 @@ SHOW BUILD INFO;
 | Field      | Description                                         |
 |------------|-----------------------------------------------------|
 | build_type | The optimization level the instance was built with. |
+
+## Active users information
+
+Running the following query will return certain information about the users that are currently logged in Memgraph,
```
[the remark this answers is not in the document]
Comment (hal-eisen-MG) [reply pages/database-management/server-stats.md:162]: Still not correct

[the remark this answers is not in the document]
Comment (Josipmrden) [reply pages/database-management/server-stats.md]: should be now I think

**Pull request merged**